### PR TITLE
[google-credentials] suggest updating Appfile with credentials JSON filepath 

### PR DESCRIPTION
**Pull request by lkesteloot**

### CODE REVIEW

**Comment by lkesteloot** [comment]
@googlebot I signed it!

File: docs/includes/google-credentials.md [hunk 1]
```diff
@@ -18,4 +18,11 @@ Tip: If you see Google Play Console or Google Developer Console in your local la
 1. Choose the permissions you'd like this account to have. We recommend **Admin (all permissions)**, but you may want to manually select all checkboxes and leave out some of the **Releases** permissions such as **Release to production**.
 1. Click **Invite user** to finish.
 
-You can use [`fastlane run validate_play_store_json_key json_key:/path/to/your/downloaded/file.json`](https://docs.fastlane.tools/actions/validate_play_store_json_key/) to test the connection to Google Play Store with the downloaded private key.
+You can use [`fastlane run validate_play_store_json_key json_key:/path/to/your/downloaded/file.json`](https://docs.fastlane.tools/actions/validate_play_store_json_key/) to test the connection to Google Play Store with the downloaded private key. Once that works, add a path to the JSON file in the `fastlane/Appfile` file:
```
Comment (rogerluan) [inline docs/includes/google-credentials.md:21]:
```suggestion
You can use [`fastlane run validate_play_store_json_key json_key:/path/to/your/downloaded/file.json`](https://docs.fastlane.tools/actions/validate_play_store_json_key/) to test the connection to Google Play Store with the downloaded private key. Once that works, add the path to the JSON file to your [Appfile](/advanced/Appfile):
```

File: docs/includes/google-credentials.md [hunk 2]
```diff
@@ -18,4 +18,11 @@ Tip: If you see Google Play Console or Google Developer Console in your local la
...
+
+```
+json_key_file("fastlane/play-store-credentials.json")
+package_name("my.package.name")
+```
```
Comment (rogerluan) [inline docs/includes/google-credentials.md:26]:
```suggestion
```ruby
json_key_file("path/to/your/play-store-credentials.json")
package_name("my.package.name")
```
```

**Review by rogerluan — CHANGES_REQUESTED** [review]
Nice addition to the docs! Just a few formatting changes 🙏

**Review by rogerluan — APPROVED** [review]
Thank you for your contribution @lkesteloot ! 💪

**Pull request merged**